swagger.yaml

```diff
@@ -21,6 +21,146 @@ paths:
             text/plain:
               schema:
                 type: string
+  /v1/projects/import:
+    post:
+      operationId: importXml
+      summary: Import XML file from Aikataulu
+      description: Import XML file from Aikataulu
+      tags:
+        - Projects
+      requestBody:
+        content:
+          application/xml:
+            schema:
+              type: string
+              format: binary
+      responses:
+        "200":
+          description: List of projects
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Project'
```

Comment:
When we Import the project - do we need to have a list returned? Just curious

Reply:
No, it's a result of failed copypaste :)

Reply:
Fixed